packages/kestrel_core/src/kestrel/interface/codegen/dataframe.py

```diff
@@ -12,10 +12,13 @@
     BoolExp,
     ExpOp,
     FExpression,
+    FBasicComparison,
+    RefComparison,
     ListOp,
     MultiComp,
     NumCompOp,
     StrCompOp,
+    AbsoluteTrue,
```

Comment:
Does `AbsoluteTrue` mean the `True` boolean literal?

Reply:
Yes, there is a chance we need empty Filter but with TimeRange:
```
x = GET event FROM schema://datalake WHERE ...
y = FIND process RESPONDED event
z = FIND process RESPONDED event START ... STOP ...
```
In these examples, `y` and `z` will have an empty `Filter` from `datalake` (a `DataSource` node) in case we need to append TimeRange (like in the case of `z`) before `ProjectEntity`.